src/redash_stmo/datasource_version.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+from redash_stmo.utils import add_resource
+
+from redash.models import DataSource
+from redash.handlers.base import BaseResource, get_object_or_404
+from redash.permissions import require_access, view_only
+
+
+DATASOURCE_VERSION_PARSE_INFO = {
+    "pg": {
+        "version_query": "select version()",
+        "delimiter": " ",
+        "index": 1
+    },
+    "redshift": {
+        "version_query": "select version()",
+        "delimiter": " ",
+        "index": -1
+    },
+    "mysql": {
+        "version_query": "select version()",
+        "delimiter": "-",
+        "index": 0
+    }
+}
+
+class DataSourceVersionResource(BaseResource):
+    def get(self, data_source_id):
+        data_source = get_object_or_404(
+            DataSource.get_by_id_and_org,
+            data_source_id,
+            self.current_org
+        )
+        require_access(data_source.groups, self.current_user, view_only)
+        try:
+            version_info = get_data_source_version(data_source.query_runner)
+        except Exception as e:
+            return {"message": unicode(e), "ok": False}
```

Comment:
I discuss below more why I don't think we'll need the the error in the frontend, but I just wanted to say here, that using `unicode` should be avoided to keep the code Python 2/3 compatible. Either way, the response probably should just look like `{'version': version_info}`.